tr/news/_posts/2020-12-25-ruby-3-0-0-released.md

```diff
@@ -0,0 +1,430 @@
+Herhangi bir geri bildirime çok ihtiyacımız var.
+
+## Diğer Dikkate Değer Yeni Özellikler
+
+* Tek satır desen eşleştirme yeniden tasarlandı. (deneysel)
+
+    * `=>` eklendi.
+      Sağ taraf ataması olarak kullanılabilir.
+
+      ```ruby
+      0 => a
+      p a #=> 0
+
+      {b: 0, c: 1} => {b:}
+      p b #=> 0
+      ```
+
+    * `in`, `true` ya da `false` döndürmesi için değiştirildi.
+
+      ```ruby
+      # sürüm 3.0
+      0 in 1 #=> false
+
+      # sürüm 2.7
+      0 in 1 #=> NoMatchingPatternError yükseltir
+      ```
+
+* Bulma deseni eklendi. (deneysel)
+
+  ``` ruby
+  case ["a", 1, "b", "c", 2, "d", "e", "f", 3]
+  in [*pre, String => x, String => y, *post]
+    p pre  #=> ["a", 1]
+    p x    #=> "b"
+    p y    #=> "c"
+    p post #=> [2, "d", "e", "f", 3]
+  end
+  ```
+
+* Sonsuz metod tanımı eklendi.
+
+  ``` ruby
+  def square(x) = x * x
+  ```
+
+* `Hash#except` şimdi gömülü.
+
+  ``` ruby
+  h = { a: 1, b: 2, c: 3 }
+  p h.except(:a) #=> {:b=>2, :c=>3}
+  ```
+
+* Hafıza görünümü deneysel bir özellik olarak eklendi.
+
+    * Bu, uzantı kütüphaneleri arasında sayısal bir dizi ve bir biteşlem görüntüsü gibi ham bir hafıza alanını takas etmek için yeni bir C-API'ıdır.
+      Uzantı kütüphaneleri ayrıca şekil, öğe biçimi, vb. içeren hafıza alanlarının üstverilerini de paylaşır.
+      Bu gibi üstverileri kullanarak, uzantı kütüphaneleri çok boyutlu dizileri bile uygun şekilde paylaşabilirler.
+      Bu özellik Python'ın tampon protokolüne danışılarak tasarlanmıştır.
+
+## Performans iyileştirmeleri
+
+* Uzun bir kodu IRB'ye yapıştırmak, Ruby 2.7.0'da gelene göre 53 kat daha hızlı.
+  Örneğin, [şu örnek kodu](https://gist.github.com/aycabta/30ab96334275bced5796f118c9220b0b) yapıştırmak için gereken zaman 11.7 saniyeden 0.22 saniyeye düşmüştür.
+
+
+<video autoplay="autoplay" controls="controls" muted="muted" width="764" height="510" poster="https://cache.ruby-lang.org/pub/media/ruby-3.0-irb-highspeed.png">
+  <source src="https://cache.ruby-lang.org/pub/media/ruby-3.0-irb-highspeed.mp4" type="video/mp4">
+</video>
+
+
+* IRB'ye `measure` komutu eklendi.
+  Bu komut basit çalıştırma zaman ölçümü sağlamaktadır.
+
+  ```
+  irb(main):001:0> 3
+  => 3
+  irb(main):002:0> measure
+  TIME is added.
+  => nil
+  irb(main):003:0> 3
+  processing time: 0.000058s
+  => 3
+  irb(main):004:0> measure :off
+  => nil
+  irb(main):005:0> 3
+  => 3
+  ```
+
+## 2.7'den bu yana diğer dikkate değer değişiklikler
+
+* Anahtar sözcük argümanları diğer argümanlardan ayrılmıştır.
+  * Prensipte, Ruby 2.7'de bir uyarı yazdıran kod çalışmayacaktır.
+    Ayrıntılar için [belgeye](https://www.ruby-lang.org/tr/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/) bakınız.
+  * Bu arada argüman yönlendirme artık sondaki argümanları da destekliyor.
+
+    ``` ruby
+    def method_missing(meth, ...)
+      send(:"do_#{ meth }", ...)
+    end
+    ```
+
+* Desen eşleştirme (`case/in`) artık deneysel değil.
```

Comment:
There is a new line below this from the English document.

https://github.com/ruby/www.ruby-lang.org/commit/70295dbb2ac6398280362aae46d7117060c262f6